activation/handler.go

```diff
@@ -233,7 +232,7 @@ func (h *Handler) getCommitmentFromAtx(atx *types.ActivationTx) ([]byte, error)
 	if err != nil {
 		return nil, err
 	}
-	return GetCommitmentBytes(atx.NodeID(), *initialATX.CommitmentATX), nil
+	return initialATX.CommitmentATX, nil
```

Comment:
How about
```suggestion
	return h.cdb.GetAtxHeader(id)
```

and remove
```go
	initialATX, err := h.cdb.GetAtxHeader(id)
	if err != nil {
		return nil, err
	}
```

Reply:
I'll add it to this PR: https://github.com/spacemeshos/go-spacemesh/pull/3788

Reply:
I'm afraid that wouldn't work. Since I don't need the (initial) ATX returned from the database, but rather just a field of it; the `CommitmentATX`.